src/main/java/org/ays/emergency_application/model/request/EmergencyEvacuationApplicationRequest.java

```diff
@@ -103,7 +101,8 @@ private boolean isPhoneNumberMustNotBeSameOne() {
             return true;
         }
 
-        if (StringUtils.isEmpty(this.applicantPhoneNumber.getLineNumber()) || StringUtils.isEmpty(this.phoneNumber.getLineNumber())) {
+        if (StringUtils.isEmpty(this.applicantPhoneNumber.getLineNumber()) ||
+                StringUtils.isEmpty(this.phoneNumber.getLineNumber())) {
```

Comment:
Burayı da eski hâline alabilir miyiz?
```suggestion
        if (StringUtils.isEmpty(this.applicantPhoneNumber.getLineNumber()) || StringUtils.isEmpty(this.phoneNumber.getLineNumber())) {
```